### PR TITLE
Experiments for reach tak in gala_kinematic

### PR DESCRIPTION
**Pull request by vincentpierre**
## Types of changes

- Docs change / refactoring / dependency upgrade
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: habitat/core/batched_env.py [hunk 1]
```diff
@@ -105,6 +105,17 @@ def get_obs(self, state):
         return _get_spherical_coordinates(ee_pos, target_pos, ee_rot)
 
 
+class RobotEESensorConfig(StateSensorConfig):
+    def __init__(self):
+        super().__init__("ROBOT_EE_RELATIVE", 3, "robot_ee_relative")
+
+    def get_obs(self, state):
```
Comment (Skylion007) [inline habitat/core/batched_env.py:112]: Do we have a type for this?

**Review by Skylion007 — APPROVED** [review]

**Review by eundersander — APPROVED** [review]
I left a comment but overall LGTM

File: habitat_baselines/config/rearrange/gala_kinematic_ddppo.yaml [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
-REWARD_SCALE: 0.01
+REWARD_SCALE: 1.0 # 0.01
 NUM_CHECKPOINTS: 0
 BATCHED_ENV: True
 OVERLAP_PHYSICS: True
-SAVE_VIDEOS_INTERVAL: 500
-NUM_UPDATES: 6100
-NUM_ENVIRONMENTS: 512
-SENSORS: ["DEPTH_SENSOR", "RGB_SENSOR", "ROBOT_START_RELATIVE", "ROBOT_TARGET_RELATIVE", "EE_START_RELATIVE", "EE_TARGET_RELATIVE", "JOINT_SENSOR"]
+SAVE_VIDEOS_INTERVAL: 50
```
[the remark this answers is not in the document]
Comment (ASzot) [reply habitat_baselines/config/rearrange/gala_kinematic_ddppo.yaml:8]: I agree with Eric, I do not think a video save interval belongs in the training loop. Instead, there should be a separate process that loads in the checkpoints and evaluates them. Then training is not slowed at all.

File: habitat/core/batched_env.py (hunk 1 above)
Comment (vincentpierre) [reply habitat/core/batched_env.py:112]: I could not find the EnvState type

File: habitat_baselines/config/rearrange/gala_kinematic_ddppo.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (vincentpierre) [reply habitat_baselines/config/rearrange/gala_kinematic_ddppo.yaml:8]:
```suggestion
SAVE_VIDEOS_INTERVAL: 500
```

**Pull request merged**